tests/server.test.js

```diff
@@ -0,0 +1,23 @@
+const axios = require('axios');
+
+describe('API Endpoint Testing', () => {
+  describe('Test /restaurants/:restaurantId/overview endpoint', () => {
+    test('should retrieve header info', async () => {
+      const { data } = await axios.get('http://localhost:8008/restaurants/30590734/overview');
```

Comment:
Nice! Just a thought: since this is hitting the DB, it's technically an integration test. If you want to run integration test separately from unit tests, you could name the file `.int.test` or the like, then have separate Jest commands that target the various file extensions. I'll look into that more, I think it'd be kinda nice separation.